test/relayed_owned_set.js

```diff
@@ -13,10 +13,9 @@ contract("TestRelayedOwnedSet", accounts => {
   let _relaySet;
   const relaySet = async () => {
     if (_relaySet === undefined) {
-      _relaySet = await TestRelaySet.new(
-        SYSTEM,
-      );
-      await _relaySet.setRelayed((await relayedOwnedSet()).address);
+      _relaySet = await TestRelaySet.new(SYSTEM);
+      const aaa = (await relayedOwnedSet()).address;
```

Comment:
```suggestion
      const contractAddress = (await relayedOwnedSet()).address;
```

Reply:
I am sorry, I used these to debug the tests, but forgot to delete them.